bedrock/firefox/tests/test_helpers.py

```diff
@@ -52,6 +52,7 @@ def check_dumb_button(self, doc):
         eq_(pq(links[5]).attr('href'), settings.APPLE_APPSTORE_FIREFOX_LINK
                                             .replace('/{country}/', '/'))
 
+    @patch('bedrock.base.waffle.switch', Mock(return_value=False))
```

Comment:
This also feels a little bit weird not having it enabled by default in dev mode. We could alternatively enable it explicitly on stage/prod. Thoughts?